CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ else()
     message(FATAL_ERROR "Unsupported CPU architecture: ${BASE_ARCHITECTURE}")
 endif()
 
+if (APPLE AND ARCHITECTURE STREQUAL "x86_64")
+    # Exclude ARM homebrew path to avoid conflicts when cross compiling.
+    set(CMAKE_IGNORE_PREFIX_PATH "${CMAKE_IGNORE_PREFIX_PATH};/opt/homebrew")
```

Comment:
`list(APPEND CMAKE_IGNORE_PREFIX_PATH "/opt/homebrew")` is more adapted for this ;)

Reply:
Done